src/cli/shared/Cicsplex.option.ts

```diff
@@ -0,0 +1,28 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright IBM Corp, 2019
+*
+*/
+
+import { ICommandOptionDefinition } from "@brightside/imperative";
+
+const MAX_LENGTH = 8;
+
+/**
+ * Imperative option for the "cicsplex" parameter
+ *
+ */
+export const CicsplexOption: ICommandOptionDefinition = {
+    name: "cicsplex",
```

Comment:
I'm not sure if you need to explicitly define this option as well as defining it in the profile. Looking at the CICS plugin, https://github.com/zowe/zowe-cli-cics-plugin/blob/master/src/cli/define/program/Program.definition.ts does not define any options for `host`, `port` etc but they are listed in the help
If so this applies to all of the shared options

Reply:
If you don't duplicate it then the you get a worse experience for the command line help. It's quite nice having a summary of all of the options when you issue --help, rather than having to combine information from the profile's help screen together with the main help screen.